src/common/hooks/index.ts

```diff
@@ -0,0 +1,3 @@
+import useScrollToTopButton from './useScrollToTopButton';
```

Comment:
a centralized hooks file can be useful in the process of unifying our styles and components. i see there is also a useEmbed hook that ill move into this file.